---
layout: paper
title: Elucidating the factors determining P53 binding and transcriptomic response to DNA damage
authors: Varsha Sundaresan, Ying Li, Benedetto DiCiaccio, Victor T. Lin, Adrian A. Acuna Higaki, Nicole F. Pelletier, Jessica T. Cheung, Lei Zhou
year: 2018
doi: 10.1158/1538-7445.AM2018-5367
---

# Abstract

Mutations in tp53 are thought to be one of the most common mechanisms by which cancer cells evade tumor suppression. Yet the exact mechanism of tumor suppression by P53 remains to be fully understood. The transcription factor P53 is activated in response to oncogenic stresses and exerts distinct anti-proliferative functions based on the stressor and cell type. Although numerous mammalian ChIP-Seq studies have identified thousands of P53 binding sites, the functionality of these binding sites remains to be established.

To study how P53 binding following DNA damage differs between cell types, we performed comparisons between P53 ChIP-Seq data from *Drosophila* embryos at different developmental stages and a Drosophila cell line (Kc167). Differential expression analysis using RNA-Seq indicated that at an early stem cell-like developmental stage of *Drosophila* embryos there is P53-dependent induction of pro-apoptotic genes in response to irradiation but not in the later differentiated stages. We aim to establish functionally significant P53 binding sites by combining ChIP-Seq and RNA-Seq data from *Drosophila* embryos as well as Kc167. The functional significance of these binding sites will be identified by CRISPR-Cas9â€“mediated genome editing. Preliminary genome-wide motif analysis studies also exhibited that homotypic clustering of P53 consensus motifs is associated with P53 binding. We will also perform comparison studies of DNA damage-induced P53 binding in humans, mice and flies to identify analogous patterns.

Preliminary studies revealed that a majority of the P53 binding sites are located at the repetitive regions of the Drosophila genome, especially the Long terminal repeats (LTRs) and Long interspersed nuclear elements (LINEs) as seen previously in mammals by other groups including Botcheva et al and Chang et al. P53 has been known to regulate the expression of retrotransposons and maintain genetic stability by keeping the number of repeats in control. We further aim to see if the functions carried out by P53 differ between repetitive vs. non-repetitive binding regions in Drosophila.

This comparative genomics study will help identify the functionally significant P53 binding sites thus further our understanding of tumor suppression by P53. The knowledge obtained from this study will also be valuable in devising novel strategies to restore P53 function in cancers.
